Formula/mdbm.rb

```diff
@@ -0,0 +1,28 @@
+class Mdbm < Formula
+  desc "Y! MDBM a very fast memory-mapped key/value store"
+  homepage "https://github.com/yahoo/mdbm/"
+  url "https://github.com/yahoo/mdbm/archive/v4.13.0.tar.gz"
+  sha256 "99cec32e02639048f96abf4475eb3f97fc669541560cd030992bab155f0cb7f8"
+
+  depends_on "coreutils"
+  depends_on "cppunit"
+  depends_on "make"
+  depends_on "openssl"
+  depends_on "readline"
+
+  def install
+    ENV.delete "CC"
+    ENV.delete "CXX"
+    system "make", "-j8"
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    ts_mdbm = "/tmp/test.mdbm"
```

Comment:
Don't write in `/tmp`. The `test` block is executed in a temporary directory, so just write files in the current directory. (Absolute path is `testpath`, if needed.)

Reply:
I Will do fixing by according to code review